### ISSUE TITLE
Fix a11y-Issue in pFocusTrap

### ISSUE THREAD

**Issue by B34v0n** [primefaces/primeng#18495]
### Describe the bug

if you use pFocusTrap, there is an a11y-violation.
Following is the error message from axe:
```
Violation found at $('span[data-pc-section="lastfocusableelement"]')
<span class="p-hidden-accessible p-hidden-focusable" tabindex="0" role="presentation" aria-hidden="true" data-p-hidden-accessible="true" data-p-hidden-focusable="true" data-pc-section="lastfocusableelement"></span>
Received:
ARIA hidden element must not be focusable or contain focusable elements (aria-hidden-focus)
Fix all of the following:
  Focusable content should have tabindex="-1" or be removed from the DOM
You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.10/aria-hidden-focus?application=axeAPI
────────
```

so there is an element, that is hidden (aria-hidden="true") and tabbable (tabindex="0").
Both together is a violation.

### Reason for not contributing a PR

- [x] Lack of time
- [x] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/focustrap#basic

### Environment

All Environments

### Angular version

19.2.9

### PrimeNG version

v19

### Node version

20.15.0

### Steps to reproduce the behavior

1. Go To https://primeng.org/focustrap#basic
2. open console
3. type `$('span[data-pc-section="lastfocusableelement"]')`
4. inspect found element
5. see `aria-hidden="true"` with `tabindex="0"`

### Expected behavior

There should not be a hidden Element that is tabbable.

**Comment by jase88** [comment]
Violation against the [4th aria rule](https://www.w3.org/TR/using-aria/#4thrule)

> Do not use _role="presentation"_ or _aria-hidden="true"_ on a __focusable element__.

**Comment by B34v0n** [comment]
This issue is still relevant
